coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,19 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
+        for observed, expected in zip_longest(
+                observed_result, expected_result):
```

Comment:
When we have a list like ^^, `self.assertEqual(observed_result, expected_result)` line inside our function gets called. I've tested it manually : 
```
AssertionError: Lists differ: ['a'] != ['a', None]

Second list contains 1 additional elements.
First extra element 1:
None

- ['a']
+ ['a', None]
```


Also, we have taken care of cases where we have `self.assertComparableObjectsEqual([Result object..., None], [Result object...])`
```
self.assertComparableObjectsEqual([Result.from_values('StylintBear', message='a', file='xyz', line=2, column=6), None], [Result.from_values('StylintBear', message='a', file='xyz', line=2, column=6)])
```
for which we get an error message as : 
```
AssertionError: Lists differ: [<Result object(id=0x377292abafc14e5bbac9614d8db12536, origin=[345 chars]None] != [<Result object(id=0x9a39c2e5f3bb4ba78d7d41e4353ff798, origin=[339 chars]2b0>]

First list contains 1 additional elements.
First extra element 1:
None

Diff is 1647 characters long. Set self.maxDiff to None to see it.
```